bin/Powheg/run_pwg.py

```diff
@@ -595,33 +600,33 @@ def runGetSource(parstage, xgrid, folderName, powInputName, process, tagName) :
 
 <initrwgt>
 <weightgroup name='nnl'> 
-<weight id='nnlops-11-1'> combines 'nn-mtmb-11' with '1001' </weight> 
-<weight id='nnlops-11-2'> combines 'nn-mtmb-11' with '1002' </weight> 
-<weight id='nnlops-11-3'> combines 'nn-mtmb-11' with '1003' </weight> 
-<weight id='nnlops-11-4'> combines 'nn-mtmb-11' with '1004' </weight> 
-<weight id='nnlops-11-5'> combines 'nn-mtmb-11' with '1005' </weight> 
-<weight id='nnlops-11-6'> combines 'nn-mtmb-11' with '1006' </weight> 
-<weight id='nnlops-11-7'> combines 'nn-mtmb-11' with '1007' </weight> 
-<weight id='nnlops-11-8'> combines 'nn-mtmb-11' with '1008' </weight> 
-<weight id='nnlops-11-9'> combines 'nn-mtmb-11' with '1009' </weight> 
-<weight id='nnlops-22-1'> combines 'nn-mtmb-22' with '1001' </weight>          
-<weight id='nnlops-22-2'> combines 'nn-mtmb-22' with '1002' </weight>         
-<weight id='nnlops-22-3'> combines 'nn-mtmb-22' with '1003' </weight>         
-<weight id='nnlops-22-4'> combines 'nn-mtmb-22' with '1004' </weight>         
-<weight id='nnlops-22-5'> combines 'nn-mtmb-22' with '1005' </weight>         
-<weight id='nnlops-22-6'> combines 'nn-mtmb-22' with '1006' </weight>         
-<weight id='nnlops-22-7'> combines 'nn-mtmb-22' with '1007' </weight>         
-<weight id='nnlops-22-8'> combines 'nn-mtmb-22' with '1008' </weight>         
-<weight id='nnlops-22-9'> combines 'nn-mtmb-22' with '1009' </weight>
-<weight id='nnlops-0505-1'> combines 'nn-mtmb-0505' with '1001' </weight>
-<weight id='nnlops-0505-2'> combines 'nn-mtmb-0505' with '1002' </weight>
-<weight id='nnlops-0505-3'> combines 'nn-mtmb-0505' with '1003' </weight>
-<weight id='nnlops-0505-4'> combines 'nn-mtmb-0505' with '1004' </weight>
-<weight id='nnlops-0505-5'> combines 'nn-mtmb-0505' with '1005' </weight>
-<weight id='nnlops-0505-6'> combines 'nn-mtmb-0505' with '1006' </weight>
-<weight id='nnlops-0505-7'> combines 'nn-mtmb-0505' with '1007' </weight>
-<weight id='nnlops-0505-8'> combines 'nn-mtmb-0505' with '1008' </weight>
-<weight id='nnlops-0505-9'> combines 'nn-mtmb-0505' with '1009' </weight>
+<weight id='nnlops-11-1'> combines 'nn-mtmb-11' with '1000' </weight> 
```

Comment:
@yuanchao 
In this script (written by you)
https://github.com/cms-sw/genproductions/blob/powheg_gcc530/bin/Powheg/make_rwl.py
the weights start all from a round number (e.g. 1000, 2000 etc.) because the counter is increased at the end of the loop. While in the past they used to start from a round number + 1. Did you mean to make this change? Otherwise, please modify the script and I will eliminate the shift from NNLOPS weights.